addons/web/static/src/search/search_bar/search_bar.js

```diff
@@ -220,6 +266,27 @@ export class SearchBar extends Component {
         return subItems;
     }
 
+    /**
+     * Compute the properties items, only when we unfold the properties field
+     * (will fetch the properties definition on the parent to generate the items).
+     *
+     * @param {Object} searchItem
+     * @param {Object} field
+     */
+    async _getPropertiesSearchItems(searchItem, field) {
+        const definitionRecord = field.definition_record;
+        const definitionRecordModel = this.fields[definitionRecord].relation;
+        const definitionRecordField = field.definition_record_field;
+
+        const propertiesSearchItems = await this.env.searchModel.getSearchItemsProperty(
+            searchItem, definitionRecord, definitionRecordModel, definitionRecordField);
+
+        const searchItems = this.searchItems.filter(
+            item => item.propertyItemId !== searchItem.id);
+
+        this.searchItems = [...searchItems, ...propertiesSearchItems];
```

Comment:
I'm guessing it would be complicated to insert them at their correct position directly, rather than sorting the array afterwards?